lib-integration-java/src/main/java/com/digitalasset/integration/internal/codec/strategies/DomXsdEncodeStrategy.java

```diff
@@ -342,7 +338,7 @@ private String printPrimitive(DamlTypes.Primitive primType, Value damlValue) {
         } else if(primType==DamlTypes.INT64) {
             return String.valueOf(damlValue.asInt64().get().getValue());
         } else if(primType==DamlTypes.DECIMAL) {
-            return damlValue.asDecimal().get().getValue().stripTrailingZeros().toPlainString();
+            return damlValue.asNumeric().get().getValue().stripTrailingZeros().toPlainString();
```

Comment:
`asDecimal` and `com.daml.ledger.javaapi.data.Decimal` are deprecated. This broke for me because the `damlValue` I was getting back was actually a `Numeric`.